Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.h

```diff
@@ -62,8 +62,16 @@ class RemoteLayerTreeDrawingAreaProxy : public DrawingAreaProxy {
     TransactionID lastCommittedLayerTreeTransactionID() const { return m_webPageProxyProcessState.transactionIDForPendingCACommit; }
 
 #if ENABLE(THREADED_ANIMATION_RESOLUTION)
-    virtual void animationsDidChangeOnNode(RemoteLayerTreeNode&) = 0;
-    virtual void updateAnimations() = 0;
+    void animationEffectStackAdded(Ref<RemoteAcceleratedEffectStack> effects);
+    void animationEffectStackRemoved(Ref<RemoteAcceleratedEffectStack> effects);
```

Comment:
I think adding a `Was` in the names would be a better fit with WebKit naming conventions.